ocpi-toolkit-2.1.1-gireve/src/main/kotlin/com/izivia/ocpi/toolkit/modules/credentials/services/CredentialsServerService.kt

```diff
@@ -38,8 +38,10 @@ class CredentialsServerService(
         tokenA: String,
         credentials: Credentials
     ): OcpiResponseBody<Credentials> = OcpiResponseBody.of {
-        val platformUrl = platformRepository.getPlatformByTokenA(tokenA)
-            ?: throw OcpiClientInvalidParametersException("Invalid CREDENTIALS_TOKEN_A ($tokenA)")
+        val platformUrl = platformRepository.savePlatformUrlForTokenA(
+            tokenA = tokenA,
+            platformUrl = credentials.url
+        ) ?: throw OcpiClientInvalidParametersException("Invalid CREDENTIALS_TOKEN_A ($tokenA)")
```

Comment:
Before the change, we needed a platformUrl associated with the tokenA. In the OCPI protocol, the receiver should only have the tokenA. Then the sender uses that tokenA and sends its information in the `Credentials` object passed in the body of the `POST /credentials` request.

This change makes sure that:
- **first**, the receiver saves the platform url given in the POST request (only if the token A is found)
- **then,** everything stays the same. We are sure that every platform object can be identified by its url (reminder: the url is the url passed  in the `Credentials` object, so it's the /versions endpoint of the sender)